tools/pip_install.py

```diff
@@ -99,8 +99,15 @@ def main(args):
         else:
             # Otherwise, we merge requirements to build the constraints and pin dependencies
             requirements = None
+            reinstall = False
             if os.environ.get('CERTBOT_OLDEST') == '1':
                 requirements = certbot_oldest_processing(tools_path, args, test_constraints)
+                # We need to --force-reinstall the tested distribution during oldest requirements
```

Comment:
Ah I see. In that case, can we just change this comment from "during oldest requirements" to "when using oldest requirements".

Reply:
Done.